RacecarSim/Assets/Scripts/Racecar/Racecar.cs

```diff
@@ -72,6 +72,11 @@ public class Racecar : MonoBehaviour
     /// The heads-up display controlled by this car, if any.
     /// </summary>
     public Hud Hud { get; set; }
+    
+    /// <summary>
+    /// Exposes the drone model.

```

Comment:
Go ahead and move this above the `Hud` so it is right after the `PhysicsModule` (to keep the modules together)